index.js

```diff
@@ -1405,9 +1405,11 @@ Client.prototype._fuzzing = function (synopsis, callback) {
 
   advance = Math.round((this.state.properties.days * msecs.day) * (1.0 - ratio))
   memo.advance1 = advance
-  if (advance > (3 * msecs.day)) advance = 3 * msecs.day
+  if (advance > (2 * msecs.day)) advance = 2 * msecs.day
   memo.advance2 = advance
-  if (advance) this.state.reconcileStamp += advance
+  if (advance) {
+    this.state.reconcileStamp = underscore.now() + 3 * msecs.day + advance
```

Comment:
So we are looking at a max of 5 days with `3 * days + advance`? Is this what we want?

Reply:
correct max push is 5 days from now

Reply:
Sounds good